### PR TITLE
Rh user profile

### PR DESCRIPTION
**Pull request by rezahusain**

### CODE REVIEW

File: app/controllers/users_controller.rb
Comment (aizatibraimova) [inline app/controllers/users_controller.rb:1]: Great job identifying user by their username!

File: config/routes.rb
Comment (aizatibraimova) [inline config/routes.rb:1]: Good to have custom route!

**Review by aizatibraimova — APPROVED** [review]

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: Great use of hidden input!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,39 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (JoseMDO) [inline app/views/users/show.html.erb:19]: We could either remove this or add default values since it is not being used :)

File: config/routes.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@
   resources :comments
   resources :photos
   devise_for :users
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:8]: Nice custom route!

**Review by JoseMDO — APPROVED** [review]
Looks good Reza!

**Comment by FadiBaker92** [comment]
Looks good, great job Reza!

**Pull request merged**